CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 0.7.0
+
+## Features
+
+* New option (`upstream_tag_template`) was added to the configuration file
+ to support more version schemes.
+* Packit is now able to be used from the distgit repository.
+  * You need to specify `upstream_project_url` to make it work.
+* New authentication scheme is possible in the configuration file.
```

Comment:
If I was a user, this would scare the hell out of me. How about just saying that we reworked authentication configuration in the user config?

Reply:
What is the scaring word, the `scheme`? I'll try to re reword that.